python/cm/models.py

```diff
@@ -32,6 +33,38 @@
 )
 
 
+class JSONFieldFormatError(Exception):
```

Comment:
We already has custom error class AdcmEx and don't need another one. Just add new error code JSON_DB_ERROR, it would be enough. So we do not have to catch new exception through all the code.

Reply:
Ok